build.env

```diff
@@ -40,3 +40,4 @@ CHANGE_MINIKUBE_NONE_USER=true
 #CEPH_CSI_RUN_ALL_TESTS=true
 E2E_TIMEOUT=60m
 DEPLOY_TIMEOUT=10
+UPGRADE_VERSION=v2.1.2
```

Comment:
How about UPGRADE_CEPHCSI_VERSION

Reply:
The travis ci complains about line being too long.... thus I had to cut it out to `UPGRADE_VERSION`

Reply:
Okay than its fine